keras_contrib/applications/densenet.py

```diff
@@ -113,8 +118,9 @@ def DenseNet(depth=40, nb_dense_block=3, growth_rate=12, nb_filter=16, nb_layers
             img_input = input_tensor
 
```

Comment:
Add a check to see if pooling is one of ['avg', 'max', None], and raise a ValueException otherwise